treeherder/webapp/api/bugzilla.py

```diff
@@ -0,0 +1,42 @@
+import requests
+from django.conf import settings
+from rest_framework import (status,
+                            viewsets)
+from rest_framework.decorators import list_route
+from rest_framework.response import Response
+
+from treeherder.etl.common import make_request
+
+
+class BugzillaViewSet(viewsets.ViewSet):
+    permission_classes = (IsAuthenticatedOrReadOnly,)
+
+    @list_route(methods=['post'])
+    def create_bug(self, request):
+        """
+        Create a bugzilla bug with passed params
+        """
+
+        params = request.data
+        url = settings.BZ_API_URL + "/rest/bug"
+        headers = {
+            'X_BUGZILLA_API_KEY': settings.BZ_API_KEY
```

Comment:
Whilst either will work, I'd recommend using the hyphenated format, per:
http://bugzilla.readthedocs.org/en/latest/api/core/v1/general.html#authentication

ie:
`X-BUGZILLA-API-KEY`